test/unit/CMakeLists.txt

```diff
@@ -129,8 +129,8 @@ create_unit_test(PREFIX datetime
                  LIBRARIES tzcode core cdt unit
 )
 create_unit_test(PREFIX error
-                 SOURCES error.c core_test_utils.c
-                 LIBRARIES unit core box_error
+                 SOURCES error.c box_test_utils.c
+                 LIBRARIES unit box core box_error
```

Comment:
Please consider adding the `tuple` library instead of `box`.

Reply:
Done. Now it looks like:

```diff
 create_unit_test(PREFIX error
                  SOURCES error.c core_test_utils.c
-                 LIBRARIES unit core box_error
+                 LIBRARIES unit core tuple box_error
 )
```